### ISSUE TITLE
[App Repo Request] Chromium

### ISSUE THREAD

**Issue by Kingdude** [ValveSoftware/SteamOS#377]
Currently to my knowledge, Chromium is not available in the app repository, and even though Steam has its own internet browser it does not have functionality to download files.

**Comment by ProfessorKaos64** [comment]
You are correct, however Valve is unlikely to add this, as IceWeasel is the target for some underpinnings here, not Chromium. I am not sure if it has to do with licensing or something akin to that. You can install firefox/chromium/chrome yourself in desktop mode. Launching this out of big picture mode is trickier. I have a github repo and scripts to achieve this, but you must be aware of the dangers, and comfortable with Linux. I would also try to be patient until November, as there may be a nice surprise for the official launch. Who knows :)

**Comment by Nyamiou** [comment]
Having the required dependency to install Google Chrome (not Chromium but the dependency are the same) could be a good idea since it's the easiest way to watch Netflix on Linux (with Firefox you have to install Pipelight).

**Comment by ProfessorKaos64** [comment]
You can install Chrome without Debian sources. I want to say there is a post on Steam Universe about it. In fact, another user previous added [this](https://github.com/ValveSoftware/SteamOS/wiki/Installing-Google-Chrome-for-Netflix) wiki page here. I've been trying to update pages, but haven't retested that one. In the past I reported a missing package when you try to install it, but a simple `apt-get install -f` should resolve the dependencies. I often prefer to install and use `gdebi` to install deb packages, so dependencies are resolved automatically (if they can be). gdebi should be available from Valve's repo.

**Comment by Kingdude** [comment]
I feel for the new users who are too used to Windows to know their way around the Linux Terminal.
I've been a Linux user since before 2012, and the Terminal still clogs my brain up sometimes. 

Windows and Macintosh users kind of need to be guided, because asking them to use the terminal is like asking the pope if he'd like to smoke a cigarette

**Comment by Tele42** [comment]
As a side note, valve is using libcef - chromium embedded framework for it webpage rendering needs, not iceweasel in the steam client.

**Comment by ProfessorKaos64** [comment]
> I feel for the new users who are too used to Windows to know their way around the Linux Terminal.
I've been a Linux user since before 2012, and the Terminal still clogs my brain up sometimes.

This is _not_ intended use of SteamOS,which has it's own browser. Desktop mode is not there as the default intended experience. It's there for us who do wish to tinker with our system. You are even given a warning when you enable it. Chromium is not offered in the Valve repositories, and will likely not. Please close.
